stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/utils/Manufacturer.kt

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2014-2022 Stream.io Inc. All rights reserved.
+ *
+ * Licensed under the Stream License;
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://github.com/GetStream/stream-chat-android/blob/main/LICENSE
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.getstream.chat.android.ui.utils
+
+import android.os.Build
+import io.getstream.chat.android.core.internal.InternalStreamChatApi
+
+/**
+ * Checks whether the device manufacturer should consume long tap.
+ * Fixes issue [#3255](https://github.com/GetStream/stream-chat-android/issues/3255)
+ *
+ * @return if manufacturer should consume long tap or not.
+ */
+@InternalStreamChatApi
```

Comment:
We don't need this annotation for internal and private methods. We only need to mark public ones that were supposed to be internal but we had to make them public because they exist in a different module

Reply:
Thanks for the info, will remove it.